doc/source/tutorials.rst

```diff
@@ -0,0 +1,17 @@
+Tutorials
+=========
+
+A learning-oriented series of federated learning tutorials, the best place to start.
+
+.. toctree::
+   :maxdepth: 1
+
+   notebooks
```

Comment:
@charlesbvll @danieljanes I just build the docs, and I feel the nesting is very deep here. We would go from currently having it directly there when users open the docs to having it two levels down. I think the tutorials need to be placed higher as in directly here opposed to moving them down.

An alternative would be that the tutorials are default open.,

<img width="344" alt="image" src="https://github.com/adap/flower/assets/423981/51a8409d-59b1-43fb-b1f9-fa07941630ed">

Reply:
@tanertopal That's right, I will remove the beginner notebooks nesting but keep the quickstart one

Reply:
Is it better like this ?
<img width="289" alt="image" src="https://github.com/adap/flower/assets/9378265/fe887fab-8fe9-4ba7-9018-e95031adaafa">